pkgs/by-name/li/libresplit/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gtk3,
+  jansson,
+  luajit,
+  pkg-config,
+  unstableGitUpdater,
+  wrapGAppsHook3,
+  xxd,
+}:
+
+stdenv.mkDerivation {
+  pname = "libresplit";
+  version = "0-unstable-2024-06-05";
+
+  src = fetchFromGitHub {
+    owner = "wins1ey";
+    repo = "LibreSplit";
+    rev = "ba10e054e2cd3a96034e3a8d5758f4ad32759ff0";
+    hash = "sha256-imKzBXwyJQeChT36FuY7ihZw+IcGbjp7LoMGRy3hM/0=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    wrapGAppsHook3
+    xxd
+  ];
+
+  buildInputs = [
+    gtk3
+    jansson
+    luajit
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  passthru.updateScript = unstableGitUpdater {
+    url = "https://github.com/wins1ey/LibreSplit.git";
```

Comment:
Redundant

Reply:
https://github.com/NixOS/nixpkgs/blob/b6050e73492a4d129b5fd3cb757804a918a9665e/pkgs/common-updater/unstable-updater.nix#L84

Nice!

Should I just remove the url or is the whole updater redundant?

Reply:
Apparently just the url is redundant (I tried both update.nix and nix-update)